tests/test_cli.py

```
@@ -258,6 +258,47 @@ def test_clean(
+def test_clean_then_write_creds_file_restores_gitignore(
+    tmp_dir_cwd: Path,
+    logfire_credentials: LogfireCredentials,
+    monkeypatch: pytest.MonkeyPatch,
+) -> None:
+    monkeypatch.setattr(sys, 'stdin', io.StringIO('y'))
+    data_dir = tmp_dir_cwd / '.logfire'
+
+    logfire_credentials.write_creds_file(data_dir)
+    assert (data_dir / '.gitignore').read_text() == '*'
+
+    main(shlex.split(f'clean --data-dir {str(data_dir)}'))
```

Ruff (0.16.0)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)